styles/base.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: "Montserrat", sans-serif;
+  src: url("./Montserrat/Montserrat-VariableFont_wght.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+body {
+  font-family: "Montserrat", sans-serif;
+  scroll-behavior: smooth;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  box-sizing: border-box;
+  padding-left: 75px;
+  padding-right: 75px;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1150px;
+}
+.block {
+  min-width: 255px;
+  max-width: 500px;;
+
+}
+.header {
+  background: radial-gradient(rgb(194, 74, 126), rgb(105, 86, 235));
+  height: 364px;
```

Comment:
Lepszym rozwiązaniem jest pozwolenie, aby zawartość wypełniła element i ona określała jej wysokość - nigdy nie wiemy czy nie będzie ona większa lub mniejsza i wtedy będzie źle wyglądałą wysokość na sztywno. Jeśli potrzebujesz odstępów to robisz je przy pomocy np. `padding`.